src/views/admin/products/new.js

```diff
@@ -0,0 +1,15 @@
+const layout = require('../layout');
+const { getError } = require('../../../utilities/getErrorHelper');
+
```

Comment:
Now that you know how to do snapshot tests, you could write a snapshot test for this, but if you don't want to because there isn't much business logic or reliance on persisting state, then that's fine.